protos/NUgus.proto

```diff
@@ -1183,11 +1217,24 @@ PROTO NUgus [
                       }
                       Camera {
                         name "left_camera"
-                        translation 0.069952 0.033795 0.06488
-                        rotation -0.570515 0.570516 0.590782 -2.07437
-                        fieldOfView 1.5707
+
+                        translation 0.08893612063588988 0.03529909622045881 0.0713674563254241
+                        rotation 0.5773189906399273 -0.5955679903440563 -0.5585709909438886 2.06585
+                        fieldOfView 1.047
                         width 640
                         height 480
+                        lens Lens {
+                          center 0.5 0.5   # [0, 1] [0, 1]
+                          radialCoefficients 0 0       # any vector
+                          tangentialCoefficients 0 0       # any vector
+                        }
+                        focus Focus {
+                          focalDistance    0   # [0, inf)
+                          focalLength      0   # [0, inf)
```

Comment:
From what I've seen, focal length of 0 is not very realistic. 
https://www.lensation.de/product/BF10M19828S118C/ our lens has 1.98 mm, but I don't know how realistic this is if we can't have most of the other features of this camera (fish eye, fov 180)...